common/src/Microsoft.Azure.IIoT.Http.HealthChecks/Microsoft.Azure.IIoT.Http.HealthChecks.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+	  <GenerateDocumentationFile>true</GenerateDocumentationFile>
+	  <Description>Azure Industrial IoT health checks</Description>
+  </PropertyGroup>
+	
+  <ItemGroup>
+	<PackageReference Include="Serilog" Version="2.10.0" />
+	<PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" />
+	<PackageReference Include="Serilog.Sinks.File" Version="4.1.0" />
```

Comment:
@simonjaeger, can you please use `PackageReference Include="Serilog.Sinks.File" Version="5.0.0"` so that whole solution uses the same version ? There are 2 other projects that reference `Serilog.Sinks.File` and both of them use version `5.0.0`.

Reply:
Upgraded to `5.0.0` version.